xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1198,30 +1198,36 @@ namespace PVR
     return false;
   }
 
-  bool CPVRGUIActions::ContinueLastPlayedChannel() const
+  bool CPVRGUIActions::PlayChannelOnStartup() const
   {
-    const CFileItemPtr item(CServiceBroker::GetPVRManager().ChannelGroups()->GetLastPlayedChannel());
-    const CPVRChannelPtr channel(item ? item->GetPVRChannelInfoTag() : CPVRChannelPtr());
-    bool bWasPlaying = false;
-    if (channel)
-    {
-      // Obtain previous 'was playing on last app quit' flag and reset it, then.
-      channel->SetWasPlayingOnLastQuit(false, bWasPlaying);
```

Comment:
Please do not remove this call. The channel that was playing on app exit must be reset on Kodi startup. Otherwise we easily can end up with multiple channels flagged as "playing on app quit" in TV database.

Since you do not need the value of the flag, please insert a call `channel->SetWasPlayingOnLastQuit(false);`.